patch/backend/app/engine/vectordbs/chroma.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
can't we use chorma db from create-llama?

Reply:
It's the ChromaDB from create-llama. But unfortunately, we need to patch the get_vector_store from that file to change the behavior. So, I moved it to this file and added the qdrant code too.

Reply:
we might consider copying it from https://raw.githubusercontent.com/run-llama/create-llama/{cl-version}/templates/components/vectordbs/python/chroma/vectordb.py then